util.go

```diff
@@ -92,11 +92,11 @@ func (b *ExponentialBackoff) exponentialBackoffStrategy() time.Duration {
 	if duration > math.MaxInt64 {
 		return b.max
 	}
-	dur := time.Duration(duration)
 
 	if b.jitter {
 		duration = rand.Float64()*(duration-initial) + initial
 	}
+	dur := time.Duration(duration)
```

Comment:
@haomingzhang - please review and comment. If this change is wrong then we need to review lines 97 - 99.

Reply:
Looks good. Looks like jitter was not taking effect, when set. There should be a TC for testing expo backoff for Jitter. If not, please open an INC. We will fix it.

Reply:
Tests with jitter value set to `true` is with default values

No proper tests for jitter `on` scenario. How do I add tests?
In this case I need to use >= due to rand.Float64 returning any value from 0.0 to 1.0
Can you point me to Golang Expect documentation?

Reply:
Yes, looks like when `b.jitter==true`, `duration` is assigned, not `dur`.
If you're looking for doc for `Expect()`, it's in Gomega:
http://onsi.github.io/gomega/

Reply:
Added 2 tests to cover jitter `true` scenario, can you review and let me know what you think?

Reply:
Ok. LGTM. Unrelated I see some of the functions and structures here start with uppercase (even though they are locally used). This needs to be fixed. We will take it up and fix it.